storage/plugins/resource-sync/ResourceSyncPlugin.php

```
@@ -0,0 +1,459 @@
+        $xw->startElementNs('rs', 'md', null);
+        $xw->writeAttribute('capability', 'resourcelist');
+        $xw->writeAttribute('at', gmdate('c'));
+        $xw->endElement();
+
+        $xw->startElementNs('rs', 'ln', null);
+        $xw->writeAttribute('rel', 'up');
+        $xw->writeAttribute('href', $base . '/resync/capabilitylist.xml');
+        $xw->endElement();
+
+        foreach ($books as $book) {
+            $id        = (int) $book['id'];
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)